wagtail/utils/version.py

```diff
@@ -20,10 +20,13 @@ def get_version(version):
     return main + sub
 
 
-def get_main_version(version=None):
+def get_main_version(version=None, include_patch=True):
     """Return main version (X.Y[.Z]) from VERSION."""
     version = get_complete_version(version)
-    parts = 2 if version[2] == 0 else 3
+    if include_patch:
+        parts = 2 if version[2] == 0 else 3
```

Comment:
Alternatively, we can make it a tri-state param, with `None` as default:

- `if include_patch is None` (the default), then use the original logic (`X.Y[.Z]`, Z only shows if not zero)
- `if include_patch is False`, then always return `X.Y`
- `if include_patch is True`, then always return `X.Y.Z` even when `Z` is zero (e.g. 6.2.0)

I'm not sure when that last use case would be useful, the only place I think it can be useful is if we change the following line for `get_semver_version`:

https://github.com/wagtail/wagtail/blob/0d23f431e956f4ae48e6ba46018a53b116dbcce2/wagtail/utils/version.py#L44-L46

To use `get_main_version(version, include_patch=True)`.